app/views/taxons/show.html.erb

```diff
@@ -1,5 +1,14 @@
 <%= display_header title: taxon.title, breadcrumbs: [:taxons, taxon] %>
 
+
+<%= link_to 'Delete taxon', taxon_path(taxon.content_id),
```

Comment:
Having this inside the `display_header` block would be more in line with other apps I think.